libraries/rtos/rtx/TARGET_CORTEX_A/RTX_CM_lib.h

```diff
@@ -85,16 +85,24 @@ OS_RESULT _os_mut_wait    (uint32_t p, OS_ID mutex, uint16_t timeout) __svc_indi
 
 #if (OS_TIMERS != 0)
 #define OS_TASK_CNT (OS_TASKCNT + 1)
+#ifndef __MBED_CMSIS_RTOS_CA9
```

Comment:
With all those ifdef on top of each one, it became hard to read. Can we do something to improve it?